src/resources/app-en.json

```diff
@@ -248,5 +248,8 @@
   "individualScheduleSetup": "Individual Schedule Setup",
   "pleaseSelectAppletToSchedule": "Please select applet to schedule for the Respondent: ",
   "respondentIsAMemberOfTheDefaultSchedule": "<1>Respondent {{secretUserId}}</1> is a member of the Default Schedule within the {{appletName}} applet. Do you want to set an Individual schedule for this Respondent?",
-  "viewDataDescription": "Please select the Applet to view Respondent's data for:"
+  "viewDataDescription": "Please select the Applet to view Respondent's data for:",
+  "dataExport": "Data Export",
+  "pleaseSelectAppletToExportRespondentsData": "Please select Applet to export Respondent's data from: ",
+  "pleaseWaitForRespondentDataDownload": "Please wait for the download to complete. It could take up to 5 minutes depending on how much data has been collected. Do not navigate away from this page."
```

Comment:
so long. maybe it's ok to rename it without "please" to make a little bit shorter?

Reply:
done